src/useLatest.js

```diff
@@ -0,0 +1,12 @@
+// @flow
```

Comment:
I would gladly just install mine - https://github.com/Andarist/use-latest but got an impressions that u dont like external deps here 😉